packages/ui/src/composables/useKeyboardOnlyFocus.ts

```diff
@@ -2,11 +2,17 @@
 
 import { ref } from 'vue'
 
+export interface KeyboardFocusListeners {
```

Comment:
This also looks redundant, keyboardFocusListeners will already have same implicit type.

Reply:
Restored previous version.